README.md

```diff
@@ -8,6 +8,9 @@ Make sure you have NPM installed. If you haven't, you can install it here: https
 
 Run the command ```npm install``` to get started. 
 
+## Structure
+![Structure]('https://doc-00-0c-docs.googleusercontent.com/docs/securesc/qa58trtcs310rehpf9o800qunh7k90sq/e291e40bbk71p2u95qckdpe8qedeup8m/1581083100000/01729543009753644624/01729543009753644624/133d5VQwkSl5JhBtEVHeHJHM0oc-o2_o9?authuser=0&nonce=eddr7j7sac9b2&user=01729543009753644624&hash=6rgd6haqe30vohb87hspuc11j6bsd1i4')
```

Comment:
Om deze URL te bekijken heb je toestemming nodig (is het een plaatje?? zoja, wat ik meestal doe is dit:
zet ze in de discord in "files to keep", doe dan copy link en gebruik die link voor het plaatje. Dan werkt het